scripts/fuzz_shell.js

```diff
@@ -134,6 +134,29 @@ function logValue(x, y) {
   console.log('[LoggingExternalInterface logging ' + printed(x, y) + ']');
 }
 
+// Table get/set operations need a BigInt if the table has 64-bit indexes. This
+// adds a proper cast as needed.
+function castTableIndex(index) {
+  if (exports.table.length == 0) {
```

Comment:
I think you can just use `if (typeof exports.table.length == 'bigint')` here and skip the rest of this function